shared/test/render-dumb-sheet.js

```diff
@@ -9,62 +9,74 @@ import DumbSheetItem from '../dev/dumb-sheet/item'
 
```

Comment:
This is actually a small change. View with whitespace off: https://github.com/keybase/client/pull/4867/commits/c9af5b83abf32e73ef5c4b97dcb4bcfef66b8095?w=1